### PR TITLE
add title

### PR DESCRIPTION
**Pull request by manuelbarzi**

### CODE REVIEW

**Comment by 21Fancy** [comment]
agree to request

File: index.html [hunk 1]
```diff
@@ -1 +1 @@
-Hello, World!
+<h1>Hello, World!</h1>
```
Comment (manuelbarzi) [inline index.html:1]: debe ser h1 o h2?

Comment (21Fancy) [reply index.html:1]: do you need to specify doctype?

Comment (SusanUva) [reply index.html:1]: yes..

Comment (GioMDiaz) [reply index.html:1]: Buen trabajo!

**Review by JudithDomenech — COMMENTED** [review]
Todo correcto.

File: index.html (hunk 1 above)
Comment (AlbaCasas) [inline index.html:1]: missing line break

Comment (Jeremiaslr) [inline index.html:1]: no css code?

Comment (JudithDomenech) [reply index.html:1]: Todo correcto

**Comment by SusanUva** [comment]
Manuel is great ! you  will  be the new manager

File: index.html (hunk 1 above)
Comment (manuelbarzi) [inline index.html:1]: hello world

**Comment by PR3D4TOR** [comment]
I like the branch, its ok! XD

File: index.html (hunk 1 above)
Comment (manuelbarzi) [reply index.html:1]: > debe ser h1 o h2?

no, es h1

Comment (manuelbarzi) [reply index.html:1]: > do you need to specify doctype?

yes, you are right, will add it

Comment (manuelbarzi) [reply index.html:1]: > falta signo exclamación de apertura

ok, will take a look

**Comment by manuelbarzi** [comment]
todo ok

**Comment by 21Fancy** [comment]
Looking clean !!

**Comment by Jeremiaslr** [comment]
okey

**Comment by PR3D4TOR** [comment]
Its ok!

**Comment by JudithDomenech** [comment]
Perfecto

**Comment by SusanUva** [comment]
👍

**Comment by PauYerAvan** [comment]
perfect!

**Pull request merged**